go/exoservice/test-fixtures/ping.go

```diff
@@ -0,0 +1,29 @@
+package exoserviceTestFixtures
+
+import (
+	"fmt"
+
+	"github.com/Originate/exocom/go/exorelay"
+	"github.com/Originate/exocom/go/exoservice"
+	"github.com/Originate/exocom/go/structs"
+)
+
+// PingTextFixture is a test fixture which responds to "ping" messages with "pong" messages
+type PingTextFixture struct {
+	ReceivedMessages []structs.Message
+}
+
+// Setup creates an ExoService instance for the given config
+func (r *PingTextFixture) Setup(config exorelay.Config) *exoservice.ExoService {
+	return &exoservice.ExoService{
+		Config: config,
+		MessageHandlers: exoservice.MessageHandlerMapping{
+			"ping": func(_ structs.MessagePayload, helpers exoservice.SendHelpers) {
```

Comment:
To simplify the method arguments, we could just provide a single object here. This is the pattern that https://github.com/gin-gonic/gin and many other frameworks use:

```go
"ping": func(r exoservice.Request) {
  // request payload is r.Payload
  err := r.Reply(exorelay.MessageOptions{Name: "pong"})
  ...